ReSharper.FSharp/src/FSharp.Psi.Features/src/ParameterInfo/FSharpParameterInfo.fs

```diff
@@ -733,27 +734,14 @@ type FSharpParameterInfoContextFactory() =
         let symbol = symbolUse.Symbol
         if not (isApplicable symbol) then None else
 
-        match context.FSharpFile.GetParseAndCheckResults(true, "FSharpParameterInfoContextFactory.getMethods") with
-        | None -> None
-        | Some results ->
-
-        let referenceOwner = reference.GetElement()
-        let names = 
-            match referenceOwner with
-            | :? IFSharpQualifiableReferenceOwner as referenceOwner -> List.ofSeq referenceOwner.Names
-            | _ -> [reference.GetName()]
-
-        let identifier = referenceOwner.FSharpIdentifier
-        if isNull identifier then None else
-
         let endCoords = endOffset.ToDocumentCoords()
         let line = int endCoords.Line + 1
         let column = int endCoords.Column + 1
```

Comment:
Coordinates calculation is still duplicated here.
Will `reference.GetElement().GetDocumentEndOffset().ToDocumentCoords()` work?

Reply:
There is no duplicate here because LambdaAnalyzer calculates coordinates differently

![image](https://user-images.githubusercontent.com/26364714/185678016-1345600d-544e-4b8d-b373-96e2fa7d422e.png)

Reply:
So, the only difference is `+ 1` in column expr? That could be a bool param instead of the duplication.
Could you check if not adding `1` could also work for Parameter Info? Or check adding it here? I wouldn't expected difference here to be honest.

Reply:
The identifier part is interesting for cases when there're type arguments, though...

Reply:
> Could you check if not adding 1 could also work for Parameter Info?

There are falling tests

> Or check adding it here?

This will also result in a bug

Reply:
Thanks for checking!

That's sad. We should really find out why the results are different someday.